data_structures/linked-list/test_linked_list.py

```diff
@@ -43,14 +43,14 @@ def test_fail_search(small_ll):
     assert small_ll.find(6) is False
 
 
-# def test_append(small_ll):
-#     """Test append to end of list."""
-#     assert small_ll.append(5) == [1, 2, 3, 4, 5]
+def test_append(small_ll):
+    """Test append to end of list."""
+    assert small_ll.append(5) is not str
```

Comment:
Since `.append()` does not return anything this is not a good test. Instead you could check that `5` was added in the correct position, just like your `test_insert_first_node` test. Keep in mind that you can access all the properties on the linked list nodes, so you can chain `._next` until you get to the correct position.